Bottom: 5b1b940c7e3af6c6585ffb3943142ef4290b7585
Top:    49db9924000fc0f365dc72fb992005d961c097f4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-10 16:47:07 +0000

Refresh of w32-update-gettext-recipe-to-0

---

diff --git a/w32/recipes/gettext.recipe b/w32/recipes/gettext.recipe
index 49b35f7..b482877 100644
--- a/w32/recipes/gettext.recipe
+++ b/w32/recipes/gettext.recipe
@@ -1,8 +1,11 @@
 name=gettext
-version=0.17
+version=0.18
 release=1
 origin=gnu
-configure_opt=--enable-relocatable
+configure_opt="
+  --enable-relocatable
+  CXX=${host:+$host-g++}
+"
 
 # Solve an incompatibility with newer MinGW runtimes
 export ac_cv_func_getopt_long_only=no
